protocol/signalfx/signalfxforwarder.go

```diff
@@ -112,6 +112,15 @@ func NewSignalfxJSONForwarer(url string, timeout time.Duration,
 		},
 		TLSHandshakeTimeout: timeout,
 	}
+
+	req, _ := http.NewRequest("POST", "https://www.signalfx.com", nil)
+	if req != nil {
+		purl, _ := tr.Proxy(req)
```

Comment:
Can you also print something if you get an error?  It tells people if their PROXY env is malformed